ruleset.xml

```diff
@@ -91,10 +94,23 @@
         <severity>5</severity><!-- turned off by PSR2 -> turning on with default severity -->
     </rule>
 
-    <!--*************************************************************************************************************-->
-    <!-- ADDITIONAL SLEVOMAT SNIFFS -->
-    <!--*************************************************************************************************************-->
+    *************************************************************************************************************
+    ADDITIONAL SLEVOMAT SNIFFS
+    *************************************************************************************************************
```

Comment:
Někam zmizely otevírací `<!--` a uzavírací značky `-->` komentářů.

Reply:
@DanCharousek doplnila jsem je, je to tam? Díky